db/migrate/20230815111534_create_like.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :posts_id, null: false, foreign_key: true
+      t.references :user_id, null: false, foreign_key: true
+
+      t.timestamps
+    end
```

Comment:
- [OPTIONAL] KIndly make sure to rename the column `author` instead of `user` in order to match the provided [ERD DIAGRAM](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) and project requirement.

```suggestion
  def change
    create_table :likes do |t|
      t.references :posts_id, null: false, foreign_key: true
     `t.references :author, foreign_key: { to_table: 'users' }`

      t.timestamps
    end
```